### PR TITLE
[ICDS Dashboard] Icds-893 Mismatch of unweight data in child benefeciary report

### PR DESCRIPTION
**Pull request by challabeehyv**
##### SUMMARY
Added valid in month filter for child benefeciary report

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/tests/agg_tests/test_export_data.py [hunk 1]
```diff
@@ -336,20 +336,20 @@ def test_children_export_child_six(self):
...
+                576, 730, '78.90 %',
                 1, 23, '4.35 %',
                 4, 23, '17.39 %',
                 18, 23, '78.26 %',
                 11, 32, '34.38 %',
                 6, 32, '18.75 %',
                 15, 32, '46.88 %',
                 0, 4, '0.00 %',
-                87, 1218, '7.14%',
+                87, 1216, '7.15%',
```
Comment (Rohit25negi) [inline custom/icds_reports/tests/agg_tests/test_export_data.py]: why is this changed by factor of 2 not 1? @challabeehyv

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left a few comments. But generally looks good.

File: custom/icds_reports/tests/agg_tests/outputs/agg_awc_2017-04-01_1_sorted.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+"st1","All","All","All","All","2017-04-01",26,130,,,,,,,,,,,,,,,,,12,,338,497,,135,0,,,,,,,,50,84,609,5,11,1,0,0,0,0,0,0,0,0,0,0,2,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,9,5,7,,2,3,7,9,85,13,3,"yes",,,,,,1,1,1,2,4,10,1315,428,428,,50,84,609,19,7,19,7,7,,,,9,177,743,2,1,1,0,1,0,0,0,0,0,0,1,0,270,91,62,3,134,0,,,6,5
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/outputs/agg_awc_2017-04-01_1_sorted.csv]: done

File: custom/icds_reports/tests/agg_tests/test_export_data.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/test_export_data.py]: we have changed two months fixtures. That is why the aggregation shows two missing

File: custom/icds_reports/reports/take_home_ration.py [hunk 3]
```diff
@@ -36,6 +36,8 @@ def _format_infrastructure_data(data):
 
         query_set = TakeHomeRationMonthly.objects.filter(**filters).order_by(*order_by)
 
+        filters = [['Generated at', india_now()]]
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/take_home_ration.py]: Why is this changed? @challabeehyv

Comment (Rohit25negi) [reply custom/icds_reports/reports/take_home_ration.py]: This looks changed by mistake?

**Review by Rohit25negi — APPROVED** [review]
Looks good. Good to merged. If my question's answer is yes(which would need to revert that line change.
@challabeehyv

File: custom/icds_reports/reports/take_home_ration.py (hunk 3 above)
Comment (challabeehyv) [reply custom/icds_reports/reports/take_home_ration.py]: it is intentional. Actually there is a difference in india_now from test case to actual implementation by 1 sec. The time is getting assigned post query which is inducing some delay. So moved it a but higher to eliminate the lag

**Comment by Rohit25negi** [comment]
@challabeehyv  test case failure looks related.
can you check.

**Comment by challabeehyv** [comment]
> @challabeehyv test case failure looks related.
> can you check.

fixed in lastest master

**Review by Rohit25negi — APPROVED** [review]
